app/pods/agendas/controller.js

```diff
@@ -20,12 +27,86 @@ export default class AgendasController extends Controller.extend(DefaultQueryPar
   @action
   closeNewSessionModal() {
     this.isCreatingNewSession = false;
+    this.newMeeting.deleteRecord();
+    this.newMeeting = this.store.createRecord('meeting');
   }
 
   @action
-  successfullyAdded() {
+  async successfullyAdded() {
+    const agenda = await this.createAgenda(this.newMeeting);
+
+    const closestMeeting = await fetchClosestMeetingAndAgendaId(this.newMeeting.plannedStart);
+    const broader = await this.newMeeting.kind?.get('broader');
```

Comment:
Try to avoid usage of `get()` but explicitly await promises (even if you now it will immediately resolve because the data has already been fetched)
```javascript
const kind = await this.newMeeting.kind;
const broader = await kind?.broader;
```